vsctl/command/connector.go

```diff
@@ -155,6 +158,9 @@ func installConnectorCommand() *cobra.Command {
 			if connectorConfigFile == "" {
 				cmdFailedf(cmd, "the --config-file flag MUST be set")
 			}
+			if !IsDNS1123Subdomain(name) {
```

Comment:
why added this if branch?

Reply:
Verify the validity of the connector name and keep it consistent with the k8s pod verification rules